appinventor/blocklyeditor/media/ai2blockly.css

```diff
@@ -28,3 +28,11 @@
   fill: #ddd;
   fill-opacity: .8;
 }
+
+.blocklybackpackImage:hover {
+  cursor: pointer;
+}
+
+.blocklyZoomButton:hover {
+  cursor: pointer;
+}
```

Comment:
@ramiAbdou Code style requires that there be a blank line at the end of this file.

Reply:
@ewpatton I'm currently working on fixing the line that shows in the reset/zoom buttons, and I am trying to figure out which values affect the positioning of the image in the sprites.png image. This is becoming extremely time-consuming to test the different values as I have to wait about 1-2 minutes for each build. Is there any way to only recompile what has changed to speed up this build time?

Reply:
@ewpatton I was able to figure it out. I removed the lines and have git added the lib/blockly changes as well. Let me know if there's anything else that needs to be done.